### DISCUSSION TITLE
Any way to support multiple Twitter accounts with Next auth?

### DISCUSSION THREAD

**Discussion by rishimohan** [nextauthjs/next-auth#3525] [Help]
Hi, I'm currently using Next-auth to allow users to login using Twitter and make Twitter API requests using user's token and secret coming from Next-auth.

My current flow is, one user logs in using Next-auth `signIn` function with Twitter as provider. I want to achieve support for quick account switching. So if I'm already logged in the app using Twitter account A, I should be able to switch to account B. Just like it's there on twitter.com.

I'm wondering if it's possible to allow quick switching between two different Twitter accounts using Next-auth, or if anyone has done it before?

The closest idea I have is that Next-auth has this login with email sort of option in `signIn` function, but I'm not sure if that would work with Twitter provider. Any help or ideas would be appreciated!

**Comment by balazsorban44** [comment]
I'm not sure how it works with OAuth 1 Twitter but in Oauth 2, you can pass a `prompt: "consent"` parameter at sign-in, which forces the user to reauthenticate. You could check if there is an alternative to that in OAuth 1
